com.github.prateekmedia.appimagepool.yaml

```
@@ -0,0 +1,47 @@
+app-id: com.github.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+rename-icons: appimagepool
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
```

```suggestion
  - --device=dri
```

At best that's all you need